ASWF/CONTRIBUTING.md

```diff
@@ -0,0 +1,333 @@
+> project is legally transferred and moved to the new repo, this file will
+> replace the one at the project root.
+>
+> TO DO:
+>
+> - [ ] Update the mail list sign-up page after the mail list moves.
+> - [ ] Update the repo URL
+> - [ ] Double check the security and info email addresses.
+>
+
+Code contributions to OpenImageIO are always welcome, and [nearly 200
+people](https://github.com/OpenImageIO/oiio/blob/master/CREDITS.md) have done
+so over the years.  Please review this document to get a briefing on our
+process.
+
+
+Mail List and Slack
+-------------------
+
+Contributors should be reading the oiio-dev mail list:
+
+* [oiio-dev](https://lists.aswf.io/g/oiio-dev)
+For developers of the OpenImageIO code itself, or users who are really
+interested in the OIIO internals. This is where we mostly discuss the code
+(including bug reports), but are also happy to answer user questions about
+use or working of OIIO.
+
+You can sign up for the mail list on your own using the link above.
+
+The [ASWF Slack](https://slack.aswf.io/) has an `openimageio` channel. Sign up
+for the Slack on your own, then under "channels", select "browse channels" and
+you should see the openimageio channel (among those of the other projects and
+working groups).
+
+
+How to Ask for Help
+-------------------
+
+If you have trouble installing, building, or using OpenImageIO, but there's
+not yet a solid reason to suspect you've encountered a genuine bug, start by
+posting a question to the [oiio-dev mailing
+list](https://lists.aswf.io/g/oiio-dev).
+This is the place for questions such has "How do I...".
+
+
+Bug Reports and Issue Tracking
+------------------------------
+
+We use GitHub's issue tracking system for reporting bugs and requesting
+enhancements: https://github.com/OpenImageIO/oiio/issues
+
+**If you are merely asking a question ("how do I...")**, please do not file an
+issue, but instead ask the question on the [oiio-dev mailing
+list](https://lists.aswf.io/g/oiio-dev).
+
+If you are submitting a bug report, please be sure to note which version of
+OIIO you are using, on what platform (OS/version, which compiler you used,
+and any special build flags or other unusual environmental issues). Please
+give a specific, as-detailed-as-possible account of
+
+* what you tried (command line, source code example)
+* what you expected to happen
+* what actually happened (crash? error message? ran but didn't give the
+  correct result?)
+
+with enough detail that others can easily reproduce the problem just by
+following your instructions. Please quote the exact error message you
+received. If you are having trouble building, please post the full cmake
+output of a fresh VERBOSE=1 build.
+
+Suspected security vulnerabilities should be reported by the same process.
+
+If confidentiality precludes a public question or issue for any reason, you
+may contact us privately at [security@openimageio.org](security@openimageio.org).
```

Comment:
Who is subscribed to security@openimageio.org? Should we mention who this is? Can anyone add themselves? Is it everyone on the TSC? A subset of the TSC based on whoever volunteers to subscribe to this?

Reply:
The security email is specifically for people to report important security vulnerabilities so that we can fix them before they are discussed in public.

It's not a subscription mail list. Right now it only forwards to me. After the domain is transferred to ASWF, we will designate whoever we want (or several people) to receive it. It could be me, or Danny, or somebody else who volunteers to be the security front line responder. It should be somebody on the TSC because it's could involve sensitive information. In reality, it only gets email once every few months, usually from people who are doing fuzz testing and want CVEs entered for the problems they find.

Just recalling how we handle it for other projects, I think it's not necessary to document who specifically receives it, but we should clarify that it is only seen by one or two people on the TSC who are the minimal set of people who need to have access so they can fix critical vulnerabilities.